templates/Web/Projects/ReactDefault/frontend/package.json

```diff
@@ -3,18 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "5.11.4",
```

Comment:
```suggestion
    "@testing-library/jest-dom": "5.11.6",
```

Reply:
Same as in angular. We will update at the end of the milestone before releasing.